rpa-builder/6.7/modules/ROOT/pages/toolbox-google-workspace-download-from-google-drive.adoc

```diff
@@ -0,0 +1,168 @@
+= Download from Google Drive 
+
+Download or Export files from your Google Drive. 
+
+You can export Google Docs, Sheets, and Slides into a selection of supported formats including PDF.
+
+== Before You Begin 
+
+Before downloading a file, ensure you: 
+
+* Configured your Google API Service Account Credentials.
++
+You can configure your credentials as an xref:toolbox-variable-handling-activity-parameters.adoc#defining-activity-parameters[Activity Parameter] or inline in the action step wizard. 
+* Obtained the corresponding File ID by using the Search in Google Drive action step. 
+
+== Properties 
+
+* *Authentication Settings* 
+** *Auth Provider x509 Cert URL*
++
+Authentication provider certificate URL for the Google API credentials. 
+** *Auth URI* 
++
+OAuth2 authentication URI for the Google API credentials.
+** *Client Email* 
++
+Client email of the Google API credentials. 
+** *Client ID* 
++
+Client ID of the Google API credentials. 
+** *Client x509 Cert URL* 
++
+Client certificate URL for the Google API credentials. 
+** *Credential Type*
++
+Type of Google API credentials. For example, `service_account`.
+** *Google API Service Account Credentials* 
++
+Activity parameter containing all Google API credentials. 
+** *Project ID* 
++
+Project ID of the Google API credentials.
+** *Token URI* 
++
+OAuth2 token URI for the Google API credentials.
+** *Universe Domain* 
++
+Universe domain of the Google API credentials.
+
+* *Common* 
++
+See xref:common-properties.adoc[Common Properties]
++
+** *Docs type* 
```

Comment:
I think the whole Google Docs type section should be moved behind the Operation mode, so that Google Docs, Sheets and Slides with their export formats are after each other

Reply:
Move line 54 - 64 to after line 79

Reply:
I copied the properties as they are listed in the Workflow Properties panel. I reordered them here in the docs as you suggested because it makes more sense.